torchvision/prototype/transforms/_misc.py

```diff
@@ -141,40 +141,26 @@ def extra_repr(self) -> str:
         return ", ".join([f"dtype={self.dtype}", f"types={[type.__name__ for type in self.types]}"])
 
 
-class _ClampBoundingBoxes(Transform):
-    _transformed_types = (features.BoundingBox,)
-
-    def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
-        output = F.clamp_bounding_box(inpt, format=inpt.format, image_size=inpt.image_size)
-        return features.BoundingBox.new_like(inpt, output)
-
-
-class _RemoveEmptyBoundingBoxes(Transform):
+class RemoveSmallBoundingBoxes(Transform):
     _transformed_types = (features.BoundingBox, features.SegmentationMask, features.Label, features.OneHotLabel)
 
+    def __init__(self, min_size: int = 1) -> None:
```

Comment:
This has to be a float because the underlying op uses floats:
```suggestion
    def __init__(self, min_size: float = 1.0) -> None:
```